spec/asyncBehaviors.js

```diff
@@ -824,5 +900,66 @@ describe('Deferred', function() {
             jasmine.Clock.tick(1);
             expect(notifySpy.argsForCall).toEqual([ ['B'] ]);
         });
+
+        it('Is superseded by rate-limit', function() {
+            this.restoreAfter(ko.options, 'deferUpdates');
+            ko.options.deferUpdates = true;
+
+            var data = ko.observable('A'),
+                deferredComputed = ko.computed(data),
+                dependentComputed = ko.computed(deferredComputed).extend({rateLimit: 500}),
+                notifySpy = jasmine.createSpy('notifySpy'),
+                subscription = dependentComputed.subscribe(notifySpy);
+
+            expect(dependentComputed()).toEqual('A');
+
+            data('B');
+            expect(deferredComputed()).toEqual('B');
+            expect(dependentComputed()).toEqual('A');       // rate-limited computed doesn't respond to "dirty" events
+
+            jasmine.Clock.tick(1);      // notifies "change" event to the rate-limited computed
+            expect(dependentComputed()).toEqual('B');       // updated on-demand
```

Comment:
Now that you bring this up, I can see how it would be confusing. I'll see how I could make it work the other way.